Extension/src/Debugger/nativeAttach.ts

```diff
@@ -175,6 +175,10 @@ function execChildProcess(process: string, workingDirectory?: string): Promise<s
             }
 
             if (stderr && stderr.length > 0) {
+                if (stderr.indexOf('screen size is bogus') >= 0) {
```

Comment:
I don't understand how this fixes it. Should there some code in the "if" statement?

Reply:
Or put the "reject" line in an "else".

Reply:
putting the reject statement inside the “else” statement was the intention, i believe i must have committed a wrong version. will fix it.